### PR TITLE
Fix wrong name for Uyuni channel

### PR DESCRIPTION
**Pull request by Bischoff**
## What does this PR change?

Fix wrong name for Uyuni channel

Follow-up of a3cbe47bff3f63335f196dd774aed8f6a8645f30

## Links

Port(s):
 * 4.3:

## Changelogs

- [x] No changelog needed

### CODE REVIEW

**Review by nodeg — APPROVED** [review]
Thank you, Eric.

File: testsuite/features/secondary/allcli_software_channels.feature [hunk 1]
```diff
@@ -80,7 +80,7 @@ Feature: Channel subscription via SSM
     And I wait until I do not see "Loading..." text
     Then radio button "openSUSE Leap 15.5 (x86_64)" should be checked
     And I wait until I do not see "Loading..." text
-    And I should see "Uyuni Proxy Devel for openSUSE Leap 15.5 (x86_64) (Development)" as unchecked
+    And I should see "Uyuni Proxy Stable for openSUSE Leap 15.5 (x86_64)" as unchecked
```
Comment (mcalmer) [inline testsuite/features/secondary/allcli_software_channels.feature:83]: Are you sure? "Stable" is what is released. But we should test development version, right?

**Review by mcalmer — CHANGES_REQUESTED** [review]

File: testsuite/features/secondary/allcli_software_channels.feature (hunk 1 above)
Comment (mcalmer) [reply testsuite/features/secondary/allcli_software_channels.feature:83]: Btw: the name come from spacewalk-common-channels.ini and `Uyuni Proxy Devel for openSUSE Leap 15.5 (x86_64) (Development)` is still a valid name there

Comment (mcalmer) [reply testsuite/features/secondary/allcli_software_channels.feature:83]: Stable: https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Proxy-POOL-x86_64-Media1/
Devel: https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master/images/repo/Uyuni-Proxy-POOL-x86_64-Media1/

I think when we want to run a test, we should test "Master"

Comment (Bischoff) [reply testsuite/features/secondary/allcli_software_channels.feature:83]: No I am not sure. @nodeg , do you have more information? Maybe we should have fixed this somehow else?

Comment (nodeg) [reply testsuite/features/secondary/allcli_software_channels.feature:83]: Yes, it makes sense to test Master (`devel`) in the CI. We identified the issue and I will prepare a fix. So we can close this PR then.

**Review by nodeg — CHANGES_REQUESTED** [review]
This PR can be closed. The new PR with the revert is https://github.com/uyuni-project/uyuni/pull/8327

File: testsuite/features/secondary/allcli_software_channels.feature (hunk 1 above)
Comment (Bischoff) [reply testsuite/features/secondary/allcli_software_channels.feature:83]: your PR is fine, closing this. Thanks a zillion @mcalmer for raising the alarm bell

**Pull request closed without merging**